src/Client.js

```diff
@@ -678,6 +678,17 @@ class Client extends EventEmitter {
         }, chatId);
     }
 
+    /**
+     * Mark Unread the Chat
+     * @param {string} chatId ID of the chat that will be unread
+     */
+    async markUnread(chatId) {
+        await this.pupPage.evaluate(async chatId => {
+            let chat = await window.Store.Chat.get(chatId);
+            await window.Store.Cmd.markChatUnread(chat, -1);
```

Comment:
Where did you get this `-1` value from? In the code as far as I can tell, it seems to send `true` in this param.

Reply:
When I run the getchats method I get the value of unread as -1 in conversations that have been marked as unread. Ex: A conversation that has no unread messages and the user marks the conversation as unread, the value that is assigned to the unreadcount is -1, since all messages have already been read, the user that marked as unread.